bbc/simorgh
# WSTEAM1-1300 - Article election banner

Resolves https://jira.dev.bbc.co.uk/browse/WSTEAM1-1300

Overall changes
======
- Adds election banner component to Article pages when certain `aboutTags` are in the article metadata
- The election banner component renders a VJ iframe to display the content on both canonical and AMP
- **This is restricted to the Test environment**

**Note** - Design, spacing etc is not yet final

Testing
======
1. Visit http://localhost.bbc.com:7080/mundo/articles/c206j730722o?renderer_env=live
2. Confirm the banner renders above the content
3. Visit http://localhost.bbc.com:7080/mundo/articles/c206j730722o.amp?renderer_env=live (amp version)
4. Confirm the banner renders above the content
5. Visit http://localhost.bbc.com:7080/mundo/articles/c2dp89lgnw7o?renderer_env=live
6. Confirm the banner does not render as this article doesn't have the correct `aboutTags`

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
